includes/Server.hpp

```diff
@@ -47,27 +52,31 @@ typedef std::map<int, Client> clientsMap;
 typedef std::map<std::string, Channel> channelsMap;
 typedef std::vector<std::string> stringVector;
 typedef std::pair<std::vector<std::string>, std::vector<std::string> >
-    pairOfStringVectors;
+    KeyValuePairList;
 
 extern Config *gConfig;
 
-enum Command {
-  JOIN,
-  KICK,
-  INVITE,
-  TOPIC,
-  MODE,
-  LIST,
-  NICK,
-  PRIVMSG,
-  QUIT,
-  PING,
-  CAP,
-  USER,
-  PASS,
-  WHO,
-  UNKNOWN
-};
+// enum Command {
+//   JOIN,
+//   KICK,
+//   INVITE,
+//   TOPIC,
+//   MODE,
+//   LIST,
+//   NICK,
+//   PRIVMSG,
+//   QUIT,
+//   PING,
+//   CAP,
+//   USER,
+//   PASS,
+//   WHO,
+//   BOT,
+//   WEATHER,
+//   TRANSLATE,
+//   ASCII_ART,
+//   UNKNOWN
+// };
```

Comment:
< Message for Melodie >
I move this enum definition to `types.hpp` to resolve an issue related to mutual inclusion between `Server class` and `Bot class`.